tests/conftest.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Setup for testing
+
+Since we're using session-scoped fixtures, declare everything in here.
+"""
+
+import pytest_asyncio
```

Comment:
```suggestion
import pytest
```

Reply:
Part of [f3a2bd5](https://github.com/ghga-de/mass/pull/9/commits/f3a2bd5831efe9215f5d521e4812b2dc84b03bef)